feat(cbh): instance resource supports startup, shutdown, and reboot actions (#4909)
huaweicloud/terraform-provider-huaweicloud: docs/resources/cbh_instance.md

  -> 1. Storage expansion is a high-risk operation, with a certain risk of failure.
  <br/>2. Expansion failure may affect the usability of the instance. Please ensure to back up your data.

> * `power_action` - (Optional, String) Specifies the power action to be done for the CBH instance.

Reviewer: ```suggestion
* `power_action` - (Optional, String) Specifies the power action after the CBH instance is created.
```

  The valid values are as follows:
  + **start**: Startup instance.
  + **stop**: Shutdown instance.

>   + **SOFT**: Normal restart, shut down virtual machine service.

Reviewer: rename **SOFT** and **HARD**.

  + **HARD**: Force restart, restart virtual machine.

>   -> The `power_action` is a one-time action.

Reviewer: ```suggestion
  -> The usage of `power_action` has some limitations. The `start` operation can only be performed when the instance 
  status is `SHUTOFF`. Only when the instance status is `ACTIVE`, the `stop`, `SOFT`, and `HARD` operations can be 
  performed.

```
